### PR TITLE
fix multi select error never shown

### PR DESCRIPTION
**Pull request by linchen2chris**
if a field is array of string/number, when submit formik will set its `isTouched`
to [], then formik-antd will set it to 'false' and its error will never shown.

### CODE REVIEW

**Comment by linchen2chris** [comment]
hi @jannikbuschke ,  I fixed a small issue for multi select error not shown when submit, hope for your approval

**Comment by wbarcovsky** [comment]
Hi @jannikbuschke 
Faced similar issue recently. Any updates about this pull request?
Happy to provide any help, if necessary

File: src/form-item/index.tsx [hunk 1]
```diff
@@ -25,14 +25,18 @@ export const FormItem = ({
...
+            ? true
+            : isTouched.reduce((acc, value) => acc || value, false)
       }
       const hasError = error !== undefined && isTouched
       const hasInitialError = initialError !== undefined
       const isValid = !error && isTouched
       const showHelp =
         hasError ||
         (hasInitialError && (!isTouched || showInitialErrorAfterTouched))
+      console.log('Line 36', showHelp, error, isTouched)
```
Comment (wbarcovsky) [inline src/form-item/index.tsx:39]: @linchen2chris Seems like debug code, that should be removed

**Comment by Maclay74** [comment]
@linchen2chris please update your merge request, this bug is annoying, let's get rid of this.

**Comment by jannikbuschke** [comment]
Ill try to close this this weekend

**Comment by jannikbuschke** [comment]
@allcontributors add linchen2chris for code

**Comment by jannikbuschke** [comment]
I used the code here in commit https://github.com/jannikbuschke/formik-antd/commit/11269b22f20663f475755f2dc18a8f3c6ed30dc2
Should be fixed in v2.0.2

**Pull request closed without merging**